src/main/java/theChillys/chillys_radio/user/controller/UserController.java

```diff
@@ -79,7 +75,11 @@ public UserResponseDto getUsersFavoriteStations(@PathVariable Long userId) {
     @GetMapping("/users/my-profile")
```

Comment:
Wrong return statement. Method must return User's profile, but returns list of favorite stations